### PR TITLE
James

### PR DESCRIPTION
**Pull request by DramaJames**
butt cheek

### CODE REVIEW

**Comment by rrahkola** [comment]
James, watch your language.

**Comment by landonDKnguyen** [comment]
you are so mature

**Comment by landonDKnguyen** [comment]
ahhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh

**Comment by DramaJames** [comment]
uwuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuuu

File: murder-mystery.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name1 = input('enter a name')
+adj1 = input('enter an adjective')
+animal1 = input('enter plural animals')
+verb1 = input('enter past tense verb')
+con1 = input('enter a conjunction')
```
Comment (rrahkola) [inline murder-mystery.py:5]: why did you want to use a conjunction here?

File: murder-mystery.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+#{con1} no matter how hard,
+#They couldn't stick him together,
+#Not even with lard.
+#""")
+
+output = "Humpty " + name1 + " sat on a wall.\n" + \
+  "Humpty " + name1 + " had a " + adj1 + " fall.\n" + \
+  "All the king's " + animal1 + " and all the king's men,\n" + \
+  "Couldn't put Humpty together again.\n" + \
+  "They " + verb1 + " and they " + verb1 + " .\n" + \
```
Comment (landonDKnguyen) [inline murder-mystery.py:23]: nice job jimmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyy

Comment (DramaJames) [reply murder-mystery.py:23]: abcdefghijklmnopqrstuvwxyz

**Comment by landonDKnguyen** [comment]
lalalalallalllalalalallalalallalalallallalalalllalallalallallalllllllllllllllllllllllllllllllllaaaaaaaaaaaaaaaaaaaaaaaallllllllllllllllllllllllllllllllllllllllaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaallllllllllllllllllllllllllllllllllllllllllllllllllaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa

**Comment by landonDKnguyen** [comment]
this is so amazing

File: murder-mystery.py (hunk 1 above)
Comment (DramaJames) [reply murder-mystery.py:5]: BUT no matter how hard

**Comment by landonDKnguyen** [comment]
how are u james

**Comment by landonDKnguyen** [comment]
hah

**Comment by DramaJames** [comment]
I'm guchi

**Pull request merged**